pysamstats/util.py

```diff
@@ -65,3 +72,11 @@ def load_stats(statfun, default_dtype, user_dtype, user_fields, **kwargs):
         a = a.view(np.recarray)
 
     return a
+
+
+def determine_max_seqid(alignmentfile):
+
+    if isinstance(alignmentfile, str):
```

Comment:
Is this ok in python 2? I noticed that you use basestring elsewhere?

Reply:
Yes I think this is OK for PY2/3 compatibility, we are trying to catch the situation where the user has put something in plain quotes, if they are using PY2 then it will be str and if PY3 then it will also be str, can ignore the fact that str in PY2 is binary data and str in PY3 is unicode data. Probably clear as mud :-)